include/rpi/hardware-control-implementation.h

```diff
@@ -50,15 +51,18 @@ class GPIO {
   // not possible.
   double StartClock(double frequency_hertz);
   void StopClock();
-
+  
```

Comment:
Don't leave trailing whitespaces in lines.
(I have just added a [`.editorconfig`](https://github.com/hzeller/txtempus/blob/master/.editorconfig) file so that it is easier to automatically do, as most editors can use that file).